app/management/clubs_list.php

```diff
@@ -3,6 +3,10 @@
     redirect("/mgmt/login");
 }
 managementPage("Clubs");
+/* Workaround to handle the case where a club is selected, so that the color works in layout.php */
+if ($club_slug = ClubManagementService::getSelectedClubSlug()) {
+    DB::setupForClub($club_slug);
```

Comment:
You are only required to do this because you rely on the singleton. As explained orally, we should try to aim at reducing singleton usage in our codebase

Reply:
I don't see why this is needed. Can you either explain or remove this part?

Reply:
Ok I think I understand why this is needed now. Nevermind you can keep it